images/win/Windows2016-Azure.json

```diff
@@ -278,6 +278,12 @@
                 "{{ template_dir }}/scripts/Installers/Install-AzureCli.ps1"
             ]
         },
+        {
+            "type": "powershell",
```

Comment:
why do we need to move Rust installation?

Reply:
there is no need in it, i reverted this change